test/specs/components/action.spec.jsx

```diff
@@ -94,6 +87,9 @@ xdescribe('Action', () => {
 
         sandbox.stub(conversationService, 'postPostback');
         conversationService.postPostback.resolves();
+
+        sandbox.stub(appUtils, 'getIntegration');
```

Comment:
I think it's a valid way to do it, we're not testing if `getIntegration` is returning the right value or not.. we want it to always return false for that test case. `getIntegration` should be tested on its own.